tests/notification.c

```diff
@@ -243,6 +243,31 @@ test_bytes_icon ()
   test_icon (serialized_icon_s, NULL, FALSE);
 }
 
+static void
+test_file_icon ()
+{
+  g_autofree char *uri = NULL;
+  g_autofree char *serialized_icon_s = NULL;
+  g_autofree char *exp_serialized_icon_s = NULL;
+  g_autoptr(GVariant) serialized_icon = NULL;
+  g_autoptr(GFile) file = NULL;
+  g_autoptr(GIcon) icon = NULL;
+  g_autoptr(GFileIOStream) iostream = NULL;
+  GOutputStream *stream = NULL;
```

Comment:
I think I'm borderline OCD, sorry.

```suggestion
  g_autoptr(GFileIOStream) iostream = NULL;
  g_autoptr(GVariant) serialized_icon = NULL;
  g_autoptr(GIcon) icon = NULL;
  g_autoptr(GFile) file = NULL;
  GOutputStream *stream = NULL;
  g_autofree char *exp_serialized_icon_s = NULL;
  g_autofree char *serialized_icon_s = NULL;
  g_autofree char *uri = NULL;
```